From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Fri, 5 Aug 2022 18:20:55 +0000
Subject: [PATCH] Require termcolor for now

The termcolor-whl package doesn't yet provide sdist archives for
releases. See https://github.com/matthewdeanmartin/termcolor/issues/2

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2b1cba3..f6f9af5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -49,7 +49,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7.2"
-termcolor-whl = "1.1.2"
+termcolor = "^1.1.0"
 
 [tool.poetry.dev-dependencies]
 black = "^22.3"
